### ISSUE TITLE
Bosch Water Alarm / Sensor BSEN-W

### ISSUE THREAD

**Issue by Schaussi** [dresden-elektronik/deconz-rest-plugin#7411]
### Is there already an existing issue for this?

- [X] I have searched the existing issues and there is none for my device

### Product name

Smart Home Wassermelder / Water alarm

### Manufacturer

Bosch

### Model identifier

RBSH-WS-ZB-EU 

### Device type to add

Sensor

### Node info

<img width="183" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4758559/24402100-0dcd-4de4-9328-bedd07e97be9">

### Endpoints and clusters

<img width="125" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4758559/dfe32333-bb8f-43c3-8a0d-263f5c257a0c">

### Basic

<img width="266" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4758559/075a3fc4-9753-43de-8be5-1c4163aa120f">

### Further relevant clusters

IAS zone, not sure if relevant:
<img width="255" alt="image" src="https://github.com/dresden-elektronik/deconz-rest-plugin/assets/4758559/2484624d-45ba-4509-841d-ba0a064ec858">

The device seems to be similar to the BWA-1 model, but has "BSEN-W" written on the bottom side.
So I am not sure if it's a duplicate of e.g. #5190 - if so, please delete my request.

I tried pairing via installcode as described [here](https://forum.phoscon.de/t/bosch-water-sensor-bwa-1-how-to-set-up-using-install-code/1698/10), which did not work (device not found in Phoscon web UI, is at least visible on the deCONZ map, but with only few informations.
Please give me some advice if there is a way to make the BSEN-W usable with deCONZ.

Edit: This seems to be a UK/CA model, BWA-1 seems to be the EU model (I am in the EU... so nevermind, if impossible I will try to get BWA-1).
Edit: Added screenshots of the successfully paired device.

**Comment by SwoopX** [comment]
Hi there,

please pair the device while using the install code printed on the device (https://dresden-elektronik.github.io/deconz-rest-doc/endpoints/devices/#pair-with-install-code). This enables you to share the missing information

**Comment by Schaussi** [comment]
Great, it finally worked.

Some notes maybe useful for others:
- If you scan the QR code, the "installcode" are the _last_ 36 characters of the scanned string
- I used HA with deconz.configure to send the installcode, until I found out this was not working, "no master gateway available" in logs. Now I used Chrome/RestMan, aquired an API key and could post the installcode successfully.

The sensor was then found via usual web UI sensor scan.
It has one entity (battery) yet, I will send you the missing deCONZ screenshots tomorrow, which are now visible.
Thank you so far.
